private/buf/cmd/buf/command/generate/generate.go

```diff
@@ -248,6 +250,12 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		"",
 		`The file or data to use for configuration.`,
 	)
+	flagSet.StringSliceVar(
+		&f.Types,
+		"types",
+		nil,
+		"The types (message, enum, service) that should be included in this image. When specified, the resulting image will only include descriptors to describe the requested types.",
```

Comment:
Need to point to types stanza in buf.gen.yaml and mention how this overrides.

Reply:
```suggestion
		"The types (message, enum, service) that should be included in this image. When specified, the resulting image will only include descriptors to describe the requested types. Flag usage overrides buf.gen.yaml",
```